website/src/community/in-person.md

```diff
@@ -28,7 +28,7 @@ While the official communication happens on the mailing list, and you can find u
 
 ## Meetups
 
-We occassionally meetup in various locations around the globe.  Active or to-be-started meetups include:
+We occasionally meetup in various locations around the globe.  Active or to-be-started meetups include:
```

Comment:
```suggestion
We occasionally meetup in various locations around the globe. Active or to-be-started meetups include:
```

Reply:
What is beam convention?  Is everything supposed to be single space?  Do we need new issues for all of the rest of site?

Reply:
OK.

Reply:
either way fine

Reply:
I think we normally use a single space after a period but this file seems to use double spaces. Fine with me.